sdk/eventhub/Microsoft.Azure.EventHubs/src/Primitives/StringUtility.cs

```diff
@@ -8,9 +8,9 @@ namespace Microsoft.Azure.EventHubs
 
     static class StringUtility
     {
-        public static string GetRandomString()
+        public static string GetRandomString(int maxLength = 6)
         {
-            return Guid.NewGuid().ToString().Substring(0, 6);
+            return Guid.NewGuid().ToString().Substring(0, maxLength);
```

Comment:
You may want to explicitly validate that `maxLength` is not greater than the length of `Guid.NewGuid().ToString()`  [36 characters], lest the caller receive an unclear exception without the right context.

Reply:
Great feedback. I will address this in the next iteration.

---
In reply to: [366007684](https://github.com/Azure/azure-sdk-for-net/pull/9443#discussion_r366007684) [](ancestors = 366007684)

Reply:
I ran all tests including Live ones. All passed. I will keep an eye on the nightly runs.